44/README.md

```diff
@@ -0,0 +1,140 @@
+1. To discuss a patch, people MAY comment on the Platform pull request, on the commit, or elsewhere.
+1. To accept or reject a patch, a Maintainer SHALL use the Platform interface.
+1. Maintainers SHOULD NOT merge their own patches except in exceptional cases, such as non-responsiveness from other Maintainers for an extended period (more than 1-2 days).
+1. Maintainers SHALL NOT make value judgments on correct patches.
+1. Maintainers SHALL merge correct patches from other Contributors rapidly.
+1. Maintainers MAY merge incorrect patches from other Contributors with the goals of (a) ending fruitless discussions, (b) capturing toxic patches in the historical record, (c) engaging with the Contributor on improving their patch quality.
+1. The user who created an issue SHOULD close the issue after checking the patch is successful.
+1. Any Contributor who has value judgments on a patch SHOULD express these via their own patches.
+1. Maintainers SHOULD close user issues that are left open without action for an uncomfortable period of time.
+
+### 2.5. Branches and Releases
+
+1. The project SHALL have one branch ("master") that always holds the latest in-progress version and SHOULD always build.
+1. The project SHALL NOT use topic branches for any reason. Personal forks MAY use topic branches.
+1. To make a stable release a Maintainer shall tag the repository. Stable releases SHALL always be released from the repository master.
+
+### 2.6. Evolution of Public Contracts
+
+1. All Public Contracts (APIs or protocols) SHALL be documented.
+1. All Public Contracts SHOULD have space for extensibility and experimentation.
+1. A patch that modifies a stable Public Contract SHOULD not break existing applications unless there is overriding consensus on the value of doing this.
+1. A patch that introduces new features SHOULD do so using new names (a new contract).
+1. New contracts SHOULD be marked as "draft" until they are stable and used by real users.
+1. Old contracts SHOULD be deprecated in a systematic fashion by marking them as "deprecated" and replacing them with new contracts as needed.
+1. When sufficient time has passed, old deprecated contracts SHOULD be removed.
+1. Old names SHALL NOT be reused by new contracts.
+1. A new contract marked as "draft" MUST not be changed to "stable" until all the following conditions are met:
+    1. Documentation has been written and finalised.
+    1. Self-tests exercising the functionality are passing on the principle target platform.
+    1. No major changes in the implementation have happened for at least one public release.
+    1. No major changes in the implementation have happened for at least 6 months.
+    1. No changes in the contract have happened for at least one public release.
+    1. No changes in the contract have happened for at least 6 months.
+    1. The developer(s) of the new contract and its implementation consent to the change of status.
```

Comment:
Who are "the developer(s)"? Those who committed the contract/implementation originally or any modifications of it?
 Those who provided feedback in some other form? What if one contributor does no longer participate, is not reachable, etc.?

At least I would change "consent" (which would require an explicit action by each developer) to "do not veto". However, some time period would need to be specified here as well. E.g. one month from raising the intent to change the status in the issue tracker?

Reply:
Perhaps I could change from "developers" to "Contributor" which is explicitly defined in 2.1.5? I'll change the consent part